### ISSUE TITLE
Add option for downloading single or multiple files from phone

### ISSUE THREAD

**Issue by mschaaf** [mkiol/SendToJolla#7]
I would like to use the app for downloading videos and images taken with the phone camera.

**Comment by mkiol** [comment]
Did you have a chance to try [AirSail Transfer](https://openrepos.net/content/6uvnpr/airsail-transfer)? It has nice GUI and works really well.

**Comment by mschaaf** [comment]
Yes, looks good and works for downloading, but it is another tool for doing almost the same. Would like to have that integrated in send to phone.

**Comment by mkiol** [comment]
Ok, thanks for your report. Fortunately, AirSail is open source, so maybe I will investigate if I can borrow some code from this project.

**Comment by mschaaf** [comment]
Thanks for considering it.

**Comment by pykong** [comment]
I second that request. I do not know whether it would be simpler to implement the capability to transfer just small files (say a couple of kb for config files) only versus large files (hundred's of MB for video files).

At least speaking from my own requirements being able to send over small files would be extremely useful on its own, especially when setting up a new phone and many config files like those for VPN need to find their way to my Jolla.

In case you do not want to add the feature in itself, maybe you could add a short note in the READM.md of this project to refer to AirSail transfer as an alternative for folks needing file transfer.
